grassroots-backend/src/grassroots-shared/Contact.entity.dto.ts

```diff
@@ -75,23 +78,34 @@ export class ContactSearchInDTO {
   @IsInt()
   @Min(1)
   id?: number;
+  @IsOptional()
   email?: string;
+  @IsOptional()
   firstName?: string;
+  @IsOptional()
   lastName?: string;
+  @IsOptional()
   phoneNumber?: string;
 }
 
 export class PaginatedContactSearchInDTO {
   @ValidateNested()
+  @Type(() => ContactSearchInDTO)
   contact!: ContactSearchInDTO;
+
   @ValidateNested()
+  @Type(() => PaginatedInDTO)
```

Comment:
Is NestJS not able to infer this from `paginated!: PaginatedInDTO`?

Reply:
These both seem like issues we'd want linting for. So easy to miss, and it seems like they'd be pretty important.

Reply:
Yup, and eslint-plugin-nestjs-typed provides rules for both of these.